pkg/controllers/vdb/restart_reconciler.go

```diff
@@ -390,6 +411,21 @@ func (r *RestartReconciler) restartCluster(ctx context.Context, downPods []*PodF
 	opts := []startdb.Option{
 		startdb.WithInitiator(r.InitiatorPod, r.InitiatorPodIP),
 	}
+	// If a communal path is set, include all of the EON parameters.
+	// Only for vclusterops as admintools relies on a config file.
```

Comment:
We are kind of breaking the abstraction by doing this. What is specific to `r.ConstructConfigParms` that doesn't work for admintools based deployments? If we do an upgrade with vclusterops enabled, will we not hit this issue?

Reply:
The issue is that at step 65, we create a vdb with a bad endpoint, so step 75 fails because we try to retrieve  the non existent s3 creds secret. With vclusterops it found that No pods qualify for possible re-ip so it did not reach the ConstructConfigParms. So the question is why is the behavior with admintools different from vclusterops'.
Should we just modify step 75 in "offline-upgrade-ks-0".

Reply:
Can we try removing the `communal.credentialSecret` parameter in 65-setup-vdb-bad-endpoint.yaml? I think at the time the testcase was made, that field was required. But it is optional now.